test/tests/functional/pbs_nodes_json.py

```diff
@@ -126,3 +126,43 @@ def test_check_reverse_solidus_escape_json(self):
             json.loads("\n".join(n_out))
         except ValueError:
             self.assertFalse(True, "Json failed to load")
+
+    def test_empty_comment_json(self):
+        """
+        Test an empty node comment (only a space) under json.
+        """
+
+        self.server.manager(MGR_CMD_SET, NODE,
+                            {'comment': ' '}, id=self.mom.shortname)
+        cmd = os.path.join(self.server.pbs_conf['PBS_EXEC'],
+                           'bin', 'pbsnodes') + ' -av -Fjson'
+        ret = self.du.run_cmd(self.server.hostname, cmd=cmd)
+        n_out = "\n".join(ret['out'])
+        self.logger.info(n_out)
+        try:
+            json.loads(n_out)
+        except ValueError:
+            self.assertFalse(True, "Json failed to load")
+
+    def test_empty_job_pset_json(self):
```

Comment:
shouldn't this be under qstat_formats? It's a qstat test.

Reply:
I was thinking since it's setting a node group key, I placed the test under pbs_nodes_json.py. But I think you're right, this should be in pbs_qstat_formats.py testsuite. I'll move it.